cmd/flags.go

```diff
@@ -47,6 +47,7 @@ func initGlobalFlags(globalFlags *pflag.FlagSet) {
 	globalFlags.StringP("engine", "e", "localhost:9076", "URL to the Qlik Associative Engine")
 	globalFlags.StringP("app", "a", "", "App name, if no app is specified a session app is used instead")
 	globalFlags.String("ttl", "30", "Qlik Associative Engine session time to live in seconds")
+	globalFlags.Bool("json", false, "Returns output in JSON format if possible, overrides the verbose and traffic flags")
```

Comment:
Technically it disables the flags, but maybe for the user it should state that verbose and traffic output is disabled

Reply:
Sounds good, updated the text now.